mlir/lib/Dialect/Rock/utility/fusionUtils.cpp

```diff
@@ -231,3 +231,24 @@ LogicalResult mlir::rock::testFusionLegalityReduce(ModuleOp mod) {
   func::FuncOp func = *(funcs.begin());
   return testFusionLegalityReduce(func);
 }
+
+LogicalResult mlir::rock::testFusionLegalityBwdConv(func::FuncOp func) {
+  WalkResult walkResult =
+      func.walk([&](rock::RockConvInterface convOp) -> WalkResult {
+        if (isa<rock::ConvBwdDataOp>(convOp) ||
+            isa<rock::ConvBwdWeightOp>(convOp)) {
+          return WalkResult::interrupt();
+        }
+        return WalkResult::advance();
+      });
+
+  return success(!walkResult.wasInterrupted());
```

Comment:
```suggestion
  return !walkResult.wasInterrupted();
```

It will get converted to success implictly.

Reply:
I don't think it will get converted implicitly. `LogicalResult` does not have a public constructor that takes a bool, so we get a compile error when trying to just return the bool.